### ISSUE TITLE
Simulations are not loading

### ISSUE THREAD

**Issue by malcolmst** [osrf/subt#976]
While checking on a few running simulations, I am currently getting an error `Currently there are no submissions` and the simulations are not loading. I've tried reloading the webpage several times, logging out and in again, and clearing cookies.

![Screenshot from 2021-07-13 08-53-44](https://user-images.githubusercontent.com/23696107/125464636-a84c25fa-d057-424e-8882-224afebde078.png)

**Comment by wolfgangschwab** [comment]
I agree. We had problems starting our simulation on cloudsim: ("LaunchingNodes Error: InitializationFailed"). And now I get the same error. So there seems to be a general problem. I wrote an email to subt-help.

**Comment by m3d** [comment]
The same with `robotika` - no submissions ...

**Comment by realdealneil** [comment]
We have the same error.  We were running three submissions last night, and they were running fine last we checked in the evening.

**Comment by nkoenig** [comment]
The service is back up. Please try and post back here if you experience any issues.

**Comment by malcolmst** [comment]
FYI it appears the same issue is happening again now

**Comment by meryl-spencer** [comment]
It is down again.

**Comment by zwn** [comment]
Well, the deadline is approaching so maybe the stress test is uncovering yet other problems (except those that were uncovered and fixed the last time).

**Comment by nkoenig** [comment]
Cloudsim seems to be working. We didn't notice an error over the past few hours. We'll continue to keep an eye on it. 

Stress is not a concern for us. Cloudsim has the capability of running hundreds of simulations simultaneously, as was done during the previous circuit events. Roughly 15 simulations have been run today (July 15th).

**Comment by nkoenig** [comment]
If you do notice an outage, can you post any additional information? For example, any error message from the REST route, if you are submitting programmatically.

**Comment by realdealneil** [comment]
I have been getting lots of "Error: Initialization Failed" errors when uploading through the webpage.  However, if Ioad the exact same set of robots/images through the REST interface, I'm having better luck.  So, I think the main problems have to do with the webpage way of loading things.  Also, the webpage keeps giving me this error:

Error #4002: Unauthorized request

So, I'm using the REST API exclusively now.

**Comment by malcolmst** [comment]
Last night I was getting “Initialization Failed” frequently too despite using the REST API. No error was returned, and it went through all the normal steps of creating nodes and pods before failing. After retrying several times (with no changes) they started running.

**Comment by malcolmst** [comment]
I’m not seeing any issues affecting the simulations once they’re running fortunately, but maybe the many users concurrently accessing the centralized part (i.e. including the web portal) could be stressing that part of the system

**Comment by m3d** [comment]
Yeah, I see the problem now again:
![Screenshot from 2021-07-19 21-20-26](https://user-images.githubusercontent.com/5664353/126215336-a281340e-ed7f-4563-bab0-32c93d0a6ef4.jpg)
(unfortunately there is no way how to bypass it - as far as I know - and download needed logfiles)

**Comment by meryl-spencer** [comment]
When using the REST API I get the following error: 'no healthy upstream'

**Comment by wolfgangschwab** [comment]
The problem is back: "Currenty there are no submissions"
And: Time out when starting a simulation via rest-service.

**Comment by wolfgangschwab** [comment]
Tried again and now it works again.

**Comment by m3d** [comment]
I am afraid it is back again :(

**Comment by m3d** [comment]
It is back now ...

**Comment by m3d** [comment]
It is broken again ... coincidentally it was at the same moment I tried to replace "Finals Preliminary Round" image, could it be related?

**Comment by m3d** [comment]
... and it is working now and submission to FPR worked.

**Comment by realdealneil** [comment]
The submission page is down for us right now (2021-08-17, 3:48 MT).  Can you reboot it?
